cms/templates/base.pt

```diff
@@ -49,7 +49,7 @@
           </div>
         </div>
 
-        <div metal:use-macro="view.search_box_template"></div>
+        <div class="base_search_box" tal:condition="not view.search_bar_404_exists()" metal:use-macro="view.search_box_template"></div>
```

Comment:
You can use `metal:define-slot="search"` and use it like how the [nav](https://github.com/universalcore/unicore-cms-ffl/blob/develop/unicorecmsffl/templates/base.pt#L58-L65) is defined. Then in the 404.pt you can just override it with an empty block. [Like so](https://github.com/universalcore/unicore-cms-ffl/blob/develop/unicorecmsffl/templates/locale_change.pt#L2)